<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessCreateForm</class>
 <widget class="QWidget" name="SessCreateForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SessCreateForm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget_Main">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_basic">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>401</width>
         <height>251</height>
        </rect>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Hostname</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtbox_hostname">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtbox_password">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtbox_username">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtbox_port">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>110</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cbox_type">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>140</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>FTP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SFTP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SCP</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Session Name</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txtbox_sessname">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>180</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_advanced">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>250</number>
      </property>
      <property name="rightMargin">
       <number>250</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_create">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
